Assets/LeapMotionModules/Interaction/Examples/Buttons/InteractionButton.cs

```diff
@@ -0,0 +1,129 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.EventSystems;
+using Leap.Unity.Attributes;
+
+namespace Leap.Unity.UI.Interaction {
+  /** A physics-enabled button. Activation is triggered by physically pushing the button back to its unsprung position. */
+  [RequireComponent(typeof(InteractionBehaviour))]
+  public class InteractionButton : MonoBehaviour {
+    //[MinMax(0f, 0.1f)]
+    [Tooltip("The minimum and maximum heights the button can exist at.")]
+    public Vector2 MinMaxHeight = new Vector2(0f, 0.02f);
+    [Tooltip("The height that this button rests at; this value is a lerp in between the min and max height.")]
+    public float RestingHeight = 0.5f;
+
+    public UnityEvent OnPress = new UnityEvent();
+    public UnityEvent OnUnpress = new UnityEvent();
+
+    [HideInInspector]
```

Comment:
HideInInspector is for when you want a variable to remain serialized, but be invisible to the user.  Seems like what you want here might be [NonSerialized]? 

Also could alternatively use 
`public bool isDepressed {get; protected set;}`
Which would make it non serialized in addition to preventing people from messing with it.